### DISCUSSION TITLE
HTTP文件服务貌似没启动

### DISCUSSION THREAD

**Discussion by TwoBall** [kkkgo/PaoPaoDNS#25] [问答和帮助]
本地访问paopao.dns:7889/ppgw.ini显示拒绝访问

**Comment by kkkgo** [comment]
贴一下启动日志

**Reply by TwoBall** [reply]
=====PaoPaoDNS docker start=====
images build time : 2023-06-16 18:56:32 UTC
New version install ! Try clean...
ulimit adbove 1000000.
              total        used        free      shared  buff/cache   available
Mem:           1998         222        1382           6         393        1699
Swap:             0           0           0
              total        used        free      shared  buff/cache   available
Mem:           2.0G      221.9M        1.3G        5.8M      393.3M        1.7G
Swap:             0           0           0
MEMSIZE:1699
====ENV TEST====
MEM:50m 100m 200 100mb
prefPC:9
CORES:1
POWCORES:2
ulimit :1073741816
FDLIM :4096
TZ:Asia/Shanghai
UPDATE:weekly
DNS_SERVERNAME:PaoPaoDNS,blog.03k.org
SERVER_IP:192.168.2.253
ETHIP:172.18.0.2
DNSPORT:53
SOCKS5:192.168.2.252:1080
CNAUTO:yes
IPV6:yes
CNFALL:yes
CUSTOM_FORWARD:192.168.2.252:53
AUTO_FORWARD:yes
AUTO_FORWARD_CHECK:yes
CN_TRACKER:yes
USE_HOSTS:no
HTTP_FILE:yes
SAFEMODE:
====ENV TEST====
Watchlist Run!
PID   USER     TIME  COMMAND
    1 root      0:00 {init.sh} /bin/sh /usr/sbin/init.sh
   18 root      0:00 crond
   66 root      0:00 dnscrypt-proxy -config /data/dnscrypt-resolvers/dnscrypt_socks.yaml
  110 root      0:00 dnscrypt-proxy -config /data/dnscrypt-resolvers/dnscrypt.toml
  111 root      0:00 unbound -c /tmp/unbound_forward.conf -p
  112 root      0:00 mosdns start -d /tmp -c mosdns.yaml
  119 root      0:00 unbound -c /tmp/unbound_raw.conf -p
  122 root      0:00 {data_update.sh} /bin/sh /usr/sbin/data_update.sh
  131 root      0:00 {watch_list.sh} /bin/sh /usr/sbin/watch_list.sh
  132 root      0:00 {watch_list.sh} /bin/sh /usr/sbin/watch_list.sh
  134 root      0:00 inotifywait -e modify,delete /etc/unbound/named.cache
  137 root      0:00 ps
  138 root      0:00 inotifywait -e modify,delete /data/Country-only-cn-private.mmdb /data/force_cn_list.txt /data/force_nocn_list.txt /data/trackerslist.txt /data/force_forward_list.txt
139:C 18 Jun 2023 12:45:57.309 # systemd supervision requested or auto-detected, but Redis is compiled without libsystemd support!
139:C 18 Jun 2023 12:45:57.311 # oO0OoO0OoO0Oo Redis is starting oO0OoO0OoO0Oo
139:C 18 Jun 2023 12:45:57.317 # Redis version=7.0.11, bits=64, commit=00000000, modified=0, pid=139, just started
139:C 18 Jun 2023 12:45:57.317 # Configuration loaded
139:M 18 Jun 2023 12:45:57.351 # Server initialized
139:M 18 Jun 2023 12:45:57.353 # WARNING Memory overcommit must be enabled! Without it, a background save or replication may fail under low memory condition. To fix this issue add 'vm.overcommit_memory = 1' to /etc/sysctl.conf and then reboot or run the command 'sysctl vm.overcommit_memory=1' for this to take effect.
fetch https://mirrors.ustc.edu.cn/alpine/edge/main/x86_64/APKINDEX.tar.gz
fetch https://mirrors.ustc.edu.cn/alpine/edge/community/x86_64/APKINDEX.tar.gz
fetch https://mirrors.nju.edu.cn/alpine/edge/main/x86_64/APKINDEX.tar.gz
fetch https://mirrors.nju.edu.cn/alpine/edge/community/x86_64/APKINDEX.tar.gz
fetch https://mirrors.aliyun.com/alpine/edge/main/x86_64/APKINDEX.tar.gz
fetch https://mirrors.aliyun.com/alpine/edge/community/x86_64/APKINDEX.tar.gz
fetch https://mirror.lzu.edu.cn/alpine/edge/main/x86_64/APKINDEX.tar.gz
fetch https://mirror.lzu.edu.cn/alpine/edge/community/x86_64/APKINDEX.tar.gz
fetch https://mirrors.tuna.tsinghua.edu.cn/alpine/edge/main/x86_64/APKINDEX.tar.gz
fetch https://mirrors.tuna.tsinghua.edu.cn/alpine/edge/community/x86_64/APKINDEX.tar.gz
fetch https://mirrors.zju.edu.cn/alpine/edge/main/x86_64/APKINDEX.tar.gz
fetch https://mirrors.zju.edu.cn/alpine/edge/community/x86_64/APKINDEX.tar.gz
fetch https://mirrors.sjtug.sjtu.edu.cn/alpine/edge/main/x86_64/APKINDEX.tar.gz
fetch https://mirrors.sjtug.sjtu.edu.cn/alpine/edge/community/x86_64/APKINDEX.tar.gz
fetch https://dl-cdn.alpinelinux.org/alpine/edge/main/x86_64/APKINDEX.tar.gz
fetch https://dl-cdn.alpinelinux.org/alpine/edge/community/x86_64/APKINDEX.tar.gz
v3.18.0-3572-gd37482ea98f [https://mirrors.ustc.edu.cn/alpine/edge/main]
v3.18.0-3576-g271c115ba89 [https://mirrors.ustc.edu.cn/alpine/edge/community]
v3.18.0-3529-gd761abc6ee7 [https://mirrors.nju.edu.cn/alpine/edge/main]
v3.18.0-3533-g6dcbfa2e2a2 [https://mirrors.nju.edu.cn/alpine/edge/community]
v3.18.0-3540-g0c0e85b5bb7 [https://mirrors.aliyun.com/alpine/edge/main]
v3.18.0-3564-ga5304169087 [https://mirrors.aliyun.com/alpine/edge/community]
v3.18.0-3572-gd37482ea98f [https://mirror.lzu.edu.cn/alpine/edge/main]
v3.18.0-3581-g923f79cc0ac [https://mirror.lzu.edu.cn/alpine/edge/community]
v3.18.0-3572-gd37482ea98f [https://mirrors.tuna.tsinghua.edu.cn/alpine/edge/main]
v3.18.0-3581-g923f79cc0ac [https://mirrors.tuna.tsinghua.edu.cn/alpine/edge/community]
v3.18.0-3570-g036195d8494 [https://mirrors.zju.edu.cn/alpine/edge/main]
v3.18.0-3569-g9d7b3c5de5b [https://mirrors.zju.edu.cn/alpine/edge/community]
v3.18.0-3572-gd37482ea98f [https://mirrors.sjtug.sjtu.edu.cn/alpine/edge/main]
v3.18.0-3581-g923f79cc0ac [https://mirrors.sjtug.sjtu.edu.cn/alpine/edge/community]
v3.18.0-3572-gd37482ea98f [https://dl-cdn.alpinelinux.org/alpine/edge/main]
v3.18.0-3581-g923f79cc0ac [https://dl-cdn.alpinelinux.org/alpine/edge/community]
OK: 20402 distinct packages available
(1/2) Upgrading libcurl (8.1.2-r0 -> 8.1.2-r1)
(2/2) Upgrading curl (8.1.2-r0 -> 8.1.2-r1)
Executing busybox-1.36.1-r1.trigger
OK: 40 MiB in 44 packages
2023-06-18 12:49:46 CST
Network error: 192.168.2.252:1080 https://www.internic.net/domain/named.cache.md5
2023-06-18 12:49:48 CST
Network error:  https://www.internic.net/domain/named.cache.md5
used_memory_human:931.65K
2023-06-18 12:49:52 CST
Network error: 192.168.2.252:1080 https://raw.githubusercontent.com/kkkgo/Country-only-cn-private.mmdb/main/Country-only-cn-private.mmdb.sha256sum
2023-06-18 12:49:55 CST
Network error:  https://raw.githubusercontent.com/kkkgo/Country-only-cn-private.mmdb/main/Country-only-cn-private.mmdb.sha256sum
2023-06-18 12:49:58 CST
Network error: 192.168.2.252:1080 https://cdn.jsdelivr.net/gh/kkkgo/Country-only-cn-private.mmdb/Country-only-cn-private.mmdb.sha256sum
2023-06-18 12:50:01 CST
Network error:  https://cdn.jsdelivr.net/gh/kkkgo/Country-only-cn-private.mmdb/Country-only-cn-private.mmdb.sha256sum
/data/trackerslist.txt MODIFY 
Trackerlist has been updated.
  112 root      0:00 mosdns start -d /tmp -c mosdns.yaml
mosdns reload...
  274 root      0:00 mosdns start -d /tmp -c mosdns.yaml
Reload watchlist...
  134 root      0:00 inotifywait -e modify,delete /etc/unbound/named.cache
Watchlist Run!
=====PaoPaoDNS docker start=====
images build time : 2023-06-16 18:56:32 UTC
ulimit adbove 1000000.
              total        used        free      shared  buff/cache   available
Mem:           1998         225        1362           6         410        1696
Swap:             0           0           0
              total        used        free      shared  buff/cache   available
Mem:           2.0G      225.4M        1.3G        5.7M      410.5M        1.7G
Swap:             0           0           0
MEMSIZE:1696
====ENV TEST====
MEM:50m 100m 200 100mb
prefPC:9
CORES:1
POWCORES:2
ulimit :1073741816
FDLIM :4096
TZ:Asia/Shanghai
UPDATE:weekly
DNS_SERVERNAME:PaoPaoDNS,blog.03k.org
SERVER_IP:192.168.2.253
ETHIP:172.18.0.2
DNSPORT:53
SOCKS5:192.168.2.252:1080
CNAUTO:yes
IPV6:yes
CNFALL:yes
CUSTOM_FORWARD:192.168.2.252:53
AUTO_FORWARD:yes
AUTO_FORWARD_CHECK:yes
CN_TRACKER:yes
USE_HOSTS:no
HTTP_FILE:yes
SAFEMODE:
====ENV TEST====
Watchlist Run!
PID   USER     TIME  COMMAND
    1 root      0:00 {init.sh} /bin/sh /usr/sbin/init.sh
   10 root      0:00 crond
   51 root      0:00 dnscrypt-proxy -config /data/dnscrypt-resolvers/dnscrypt_socks.yaml
   93 root      0:00 dnscrypt-proxy -config /data/dnscrypt-resolvers/dnscrypt.toml
   94 root      0:00 unbound -c /tmp/unbound_forward.conf -p
   95 root      0:00 mosdns start -d /tmp -c mosdns.yaml
  103 root      0:00 unbound -c /tmp/unbound_raw.conf -p
  106 root      0:00 {data_update.sh} /bin/sh /usr/sbin/data_update.sh
  115 root      0:00 ps
  116 root      0:00 {watch_list.sh} /bin/sh /usr/sbin/watch_list.sh
  117 root      0:00 {watch_list.sh} /bin/sh /usr/sbin/watch_list.sh
  118 root      0:00 [watch_list.sh]
  119 root      0:00 {watch_list.sh} /bin/sh /usr/sbin/watch_list.sh
  120 root      0:00 inotifywait -e modify,delete /etc/unbound/named.cache
122:C 18 Jun 2023 13:14:35.148 # systemd supervision requested or auto-detected, but Redis is compiled without libsystemd support!
122:C 18 Jun 2023 13:14:35.155 # oO0OoO0OoO0Oo Redis is starting oO0OoO0OoO0Oo
122:C 18 Jun 2023 13:14:35.156 # Redis version=7.0.11, bits=64, commit=00000000, modified=0, pid=122, just started
122:C 18 Jun 2023 13:14:35.161 # Configuration loaded
122:M 18 Jun 2023 13:14:35.179 # Server initialized
122:M 18 Jun 2023 13:14:35.199 # WARNING Memory overcommit must be enabled! Without it, a background save or replication may fail under low memory condition. To fix this issue add 'vm.overcommit_memory = 1' to /etc/sysctl.conf and then reboot or run the command 'sysctl vm.overcommit_memory=1' for this to take effect.
fetch https://mirrors.ustc.edu.cn/alpine/edge/main/x86_64/APKINDEX.tar.gz
fetch https://mirrors.ustc.edu.cn/alpine/edge/community/x86_64/APKINDEX.tar.gz
fetch https://mirrors.nju.edu.cn/alpine/edge/main/x86_64/APKINDEX.tar.gz
fetch https://mirrors.nju.edu.cn/alpine/edge/community/x86_64/APKINDEX.tar.gz
fetch https://mirrors.aliyun.com/alpine/edge/main/x86_64/APKINDEX.tar.gz
fetch https://mirrors.aliyun.com/alpine/edge/community/x86_64/APKINDEX.tar.gz
fetch https://mirror.lzu.edu.cn/alpine/edge/main/x86_64/APKINDEX.tar.gz
fetch https://mirror.lzu.edu.cn/alpine/edge/community/x86_64/APKINDEX.tar.gz
fetch https://mirrors.tuna.tsinghua.edu.cn/alpine/edge/main/x86_64/APKINDEX.tar.gz
fetch https://mirrors.tuna.tsinghua.edu.cn/alpine/edge/community/x86_64/APKINDEX.tar.gz
fetch https://mirrors.zju.edu.cn/alpine/edge/main/x86_64/APKINDEX.tar.gz
fetch https://mirrors.zju.edu.cn/alpine/edge/community/x86_64/APKINDEX.tar.gz
fetch https://mirrors.sjtug.sjtu.edu.cn/alpine/edge/main/x86_64/APKINDEX.tar.gz
fetch https://mirrors.sjtug.sjtu.edu.cn/alpine/edge/community/x86_64/APKINDEX.tar.gz
fetch https://dl-cdn.alpinelinux.org/alpine/edge/main/x86_64/APKINDEX.tar.gz
fetch https://dl-cdn.alpinelinux.org/alpine/edge/community/x86_64/APKINDEX.tar.gz
v3.18.0-3572-gd37482ea98f [https://mirrors.ustc.edu.cn/alpine/edge/main]
v3.18.0-3576-g271c115ba89 [https://mirrors.ustc.edu.cn/alpine/edge/community]
v3.18.0-3529-gd761abc6ee7 [https://mirrors.nju.edu.cn/alpine/edge/main]
v3.18.0-3533-g6dcbfa2e2a2 [https://mirrors.nju.edu.cn/alpine/edge/community]
v3.18.0-3540-g0c0e85b5bb7 [https://mirrors.aliyun.com/alpine/edge/main]
v3.18.0-3564-ga5304169087 [https://mirrors.aliyun.com/alpine/edge/community]
v3.18.0-3572-gd37482ea98f [https://mirror.lzu.edu.cn/alpine/edge/main]
v3.18.0-3581-g923f79cc0ac [https://mirror.lzu.edu.cn/alpine/edge/community]
v3.18.0-3572-gd37482ea98f [https://mirrors.tuna.tsinghua.edu.cn/alpine/edge/main]
v3.18.0-3581-g923f79cc0ac [https://mirrors.tuna.tsinghua.edu.cn/alpine/edge/community]
v3.18.0-3572-gd37482ea98f [https://mirrors.zju.edu.cn/alpine/edge/main]
v3.18.0-3581-g923f79cc0ac [https://mirrors.zju.edu.cn/alpine/edge/community]
v3.18.0-3572-gd37482ea98f [https://mirrors.sjtug.sjtu.edu.cn/alpine/edge/main]
v3.18.0-3581-g923f79cc0ac [https://mirrors.sjtug.sjtu.edu.cn/alpine/edge/community]
v3.18.0-3572-gd37482ea98f [https://dl-cdn.alpinelinux.org/alpine/edge/main]
v3.18.0-3581-g923f79cc0ac [https://dl-cdn.alpinelinux.org/alpine/edge/community]
OK: 20400 distinct packages available
OK: 40 MiB in 44 packages
2023-06-18 13:19:07 CST
Network error: 192.168.2.252:1080 https://www.internic.net/domain/named.cache.md5
2023-06-18 13:19:07 CST
Network error:  https://www.internic.net/domain/named.cache.md5
used_memory_human:948.89K
2023-06-18 13:19:08 CST
Network error: 192.168.2.252:1080 https://raw.githubusercontent.com/kkkgo/Country-only-cn-private.mmdb/main/Country-only-cn-private.mmdb.sha256sum
2023-06-18 13:19:08 CST
Network error:  https://raw.githubusercontent.com/kkkgo/Country-only-cn-private.mmdb/main/Country-only-cn-private.mmdb.sha256sum
2023-06-18 13:19:08 CST
Network error: 192.168.2.252:1080 https://cdn.jsdelivr.net/gh/kkkgo/Country-only-cn-private.mmdb/Country-only-cn-private.mmdb.sha256sum
2023-06-18 13:19:08 CST
Network error:  https://cdn.jsdelivr.net/gh/kkkgo/Country-only-cn-private.mmdb/Country-only-cn-private.mmdb.sha256sum
/data/trackerslist.txt MODIFY 
Trackerlist has been updated.
   95 root      0:00 mosdns start -d /tmp -c mosdns.yaml
mosdns reload...
  258 root      0:00 mosdns start -d /tmp -c mosdns.yaml
Reload watchlist...
  120 root      0:00 inotifywait -e modify,delete /etc/unbound/named.cache
Watchlist Run!
=====PaoPaoDNS docker start=====
images build time : 2023-06-16 18:56:32 UTC
ulimit adbove 1000000.
              total        used        free      shared  buff/cache   available
Mem:           1998         221        1257           6         520        1699
Swap:             0           0           0
              total        used        free      shared  buff/cache   available
Mem:           2.0G      220.7M        1.2G        5.7M      519.9M        1.7G
Swap:             0           0           0
MEMSIZE:1699
====ENV TEST====
MEM:50m 100m 200 100mb
prefPC:9
CORES:1
POWCORES:2
ulimit :1073741816
FDLIM :4096
TZ:Asia/Shanghai
UPDATE:weekly
DNS_SERVERNAME:PaoPaoDNS,blog.03k.org
SERVER_IP:192.168.2.253
ETHIP:172.18.0.2
DNSPORT:53
SOCKS5:192.168.2.252:1080
CNAUTO:yes
IPV6:yes
CNFALL:yes
CUSTOM_FORWARD:192.168.2.252:53
AUTO_FORWARD:yes
AUTO_FORWARD_CHECK:yes
CN_TRACKER:yes
USE_HOSTS:no
HTTP_FILE:yes
SAFEMODE:
====ENV TEST====
Watchlist Run!
PID   USER     TIME  COMMAND
    1 root      0:00 {init.sh} /bin/sh /usr/sbin/init.sh
   11 root      0:00 crond
   52 root      0:00 dnscrypt-proxy -config /data/dnscrypt-resolvers/dnscrypt_socks.yaml
   94 root      0:00 dnscrypt-proxy -config /data/dnscrypt-resolvers/dnscrypt.toml
   95 root      0:00 unbound -c /tmp/unbound_forward.conf -p
   96 root      0:00 mosdns start -d /tmp -c mosdns.yaml
  103 root      0:00 unbound -c /tmp/unbound_raw.conf -p
  106 root      0:00 {data_update.sh} /bin/sh /usr/sbin/data_update.sh
  115 root      0:00 {watch_list.sh} /bin/sh /usr/sbin/watch_list.sh
  116 root      0:00 {watch_list.sh} /bin/sh /usr/sbin/watch_list.sh
  118 root      0:00 inotifywait -e modify,delete /etc/unbound/named.cache
  121 root      0:00 ps
  122 root      0:00 {watch_list.sh} /bin/sh /usr/sbin/watch_list.sh
123:C 18 Jun 2023 14:08:50.120 # systemd supervision requested or auto-detected, but Redis is compiled without libsystemd support!
123:C 18 Jun 2023 14:08:50.121 # oO0OoO0OoO0Oo Redis is starting oO0OoO0OoO0Oo
123:C 18 Jun 2023 14:08:50.125 # Redis version=7.0.11, bits=64, commit=00000000, modified=0, pid=123, just started
123:C 18 Jun 2023 14:08:50.129 # Configuration loaded
123:M 18 Jun 2023 14:08:50.160 # Server initialized
123:M 18 Jun 2023 14:08:50.195 # WARNING Memory overcommit must be enabled! Without it, a background save or replication may fail under low memory condition. To fix this issue add 'vm.overcommit_memory = 1' to /etc/sysctl.conf and then reboot or run the command 'sysctl vm.overcommit_memory=1' for this to take effect.
fetch https://mirrors.ustc.edu.cn/alpine/edge/main/x86_64/APKINDEX.tar.gz
fetch https://mirrors.ustc.edu.cn/alpine/edge/community/x86_64/APKINDEX.tar.gz
fetch https://mirrors.nju.edu.cn/alpine/edge/main/x86_64/APKINDEX.tar.gz
fetch https://mirrors.nju.edu.cn/alpine/edge/community/x86_64/APKINDEX.tar.gz
fetch https://mirrors.aliyun.com/alpine/edge/main/x86_64/APKINDEX.tar.gz
fetch https://mirrors.aliyun.com/alpine/edge/community/x86_64/APKINDEX.tar.gz
fetch https://mirror.lzu.edu.cn/alpine/edge/main/x86_64/APKINDEX.tar.gz
fetch https://mirror.lzu.edu.cn/alpine/edge/community/x86_64/APKINDEX.tar.gz
fetch https://mirrors.tuna.tsinghua.edu.cn/alpine/edge/main/x86_64/APKINDEX.tar.gz
fetch https://mirrors.tuna.tsinghua.edu.cn/alpine/edge/community/x86_64/APKINDEX.tar.gz
fetch https://mirrors.zju.edu.cn/alpine/edge/main/x86_64/APKINDEX.tar.gz
fetch https://mirrors.zju.edu.cn/alpine/edge/community/x86_64/APKINDEX.tar.gz
fetch https://mirrors.sjtug.sjtu.edu.cn/alpine/edge/main/x86_64/APKINDEX.tar.gz
fetch https://mirrors.sjtug.sjtu.edu.cn/alpine/edge/community/x86_64/APKINDEX.tar.gz
fetch https://dl-cdn.alpinelinux.org/alpine/edge/main/x86_64/APKINDEX.tar.gz
fetch https://dl-cdn.alpinelinux.org/alpine/edge/community/x86_64/APKINDEX.tar.gz
v3.18.0-3572-gd37482ea98f [https://mirrors.ustc.edu.cn/alpine/edge/main]
v3.18.0-3576-g271c115ba89 [https://mirrors.ustc.edu.cn/alpine/edge/community]
v3.18.0-3572-gd37482ea98f [https://mirrors.nju.edu.cn/alpine/edge/main]
v3.18.0-3581-g923f79cc0ac [https://mirrors.nju.edu.cn/alpine/edge/community]
v3.18.0-3540-g0c0e85b5bb7 [https://mirrors.aliyun.com/alpine/edge/main]
v3.18.0-3564-ga5304169087 [https://mirrors.aliyun.com/alpine/edge/community]
v3.18.0-3572-gd37482ea98f [https://mirror.lzu.edu.cn/alpine/edge/main]
v3.18.0-3581-g923f79cc0ac [https://mirror.lzu.edu.cn/alpine/edge/community]
v3.18.0-3572-gd37482ea98f [https://mirrors.tuna.tsinghua.edu.cn/alpine/edge/main]
v3.18.0-3581-g923f79cc0ac [https://mirrors.tuna.tsinghua.edu.cn/alpine/edge/community]
v3.18.0-3572-gd37482ea98f [https://mirrors.zju.edu.cn/alpine/edge/main]
v3.18.0-3581-g923f79cc0ac [https://mirrors.zju.edu.cn/alpine/edge/community]
v3.18.0-3572-gd37482ea98f [https://mirrors.sjtug.sjtu.edu.cn/alpine/edge/main]
v3.18.0-3581-g923f79cc0ac [https://mirrors.sjtug.sjtu.edu.cn/alpine/edge/community]
v3.18.0-3572-gd37482ea98f [https://dl-cdn.alpinelinux.org/alpine/edge/main]
v3.18.0-3581-g923f79cc0ac [https://dl-cdn.alpinelinux.org/alpine/edge/community]
OK: 20333 distinct packages available
OK: 40 MiB in 44 packages
2023-06-18 14:15:27 CST
/etc/unbound/named.cache Same hash, skip update.
used_memory_human:1.01M
2023-06-18 14:15:28 CST
/data/Country-only-cn-private.mmdb Same hash, skip update.
/data/trackerslist.txt MODIFY 
Trackerlist has been updated.
   96 root      0:00 mosdns start -d /tmp -c mosdns.yaml
mosdns reload...
  209 root      0:00 mosdns start -d /tmp -c mosdns.yaml
Reload watchlist...
  118 root      0:00 inotifywait -e modify,delete /etc/unbound/named.cache
Watchlist Run!
=====PaoPaoDNS docker start=====
images build time : 2023-06-16 18:56:32 UTC
ulimit adbove 1000000.
              total        used        free      shared  buff/cache   available
Mem:           1998         271        1241           6         486        1649
Swap:             0           0           0
              total        used        free      shared  buff/cache   available
Mem:           2.0G      271.0M        1.2G        5.7M      485.5M        1.6G
Swap:             0           0           0
MEMSIZE:1649
====ENV TEST====
MEM:50m 100m 200 100mb
prefPC:9
CORES:1
POWCORES:2
ulimit :1073741816
FDLIM :4096
TZ:Asia/Shanghai
UPDATE:weekly
DNS_SERVERNAME:PaoPaoDNS,blog.03k.org
SERVER_IP:192.168.2.253
ETHIP:172.18.0.2
DNSPORT:53
SOCKS5:192.168.2.252:1080
CNAUTO:yes
IPV6:yes
CNFALL:yes
CUSTOM_FORWARD:192.168.2.252:53
AUTO_FORWARD:yes
AUTO_FORWARD_CHECK:yes
CN_TRACKER:yes
USE_HOSTS:no
HTTP_FILE:yes
SAFEMODE:
====ENV TEST====
Watchlist Run!
PID   USER     TIME  COMMAND
    1 root      0:00 {init.sh} /bin/sh /usr/sbin/init.sh
   11 root      0:00 crond
   52 root      0:00 dnscrypt-proxy -config /data/dnscrypt-resolvers/dnscrypt_socks.yaml
   94 root      0:00 dnscrypt-proxy -config /data/dnscrypt-resolvers/dnscrypt.toml
   95 root      0:00 unbound -c /tmp/unbound_forward.conf -p
   96 root      0:00 mosdns start -d /tmp -c mosdns.yaml
  101 root      0:00 unbound -c /tmp/unbound_raw.conf -p
  104 root      0:00 {data_update.sh} /bin/sh /usr/sbin/data_update.sh
  115 root      0:00 {watch_list.sh} /bin/sh /usr/sbin/watch_list.sh
  116 root      0:00 {watch_list.sh} /bin/sh /usr/sbin/watch_list.sh
  117 root      0:00 inotifywait -e modify,delete /etc/unbound/named.cache
  120 root      0:00 ps
  121 root      0:00 inotifywait -e modify,delete /data/Country-only-cn-private.mmdb /data/force_cn_list.txt /data/force_nocn_list.txt /data/trackerslist.txt /data/force_forward_list.txt
122:C 18 Jun 2023 15:06:29.071 # systemd supervision requested or auto-detected, but Redis is compiled without libsystemd support!
122:C 18 Jun 2023 15:06:29.075 # oO0OoO0OoO0Oo Redis is starting oO0OoO0OoO0Oo
122:C 18 Jun 2023 15:06:29.076 # Redis version=7.0.11, bits=64, commit=00000000, modified=0, pid=122, just started
122:C 18 Jun 2023 15:06:29.076 # Configuration loaded
122:M 18 Jun 2023 15:06:29.127 # Server initialized
122:M 18 Jun 2023 15:06:29.131 # WARNING Memory overcommit must be enabled! Without it, a background save or replication may fail under low memory condition. To fix this issue add 'vm.overcommit_memory = 1' to /etc/sysctl.conf and then reboot or run the command 'sysctl vm.overcommit_memory=1' for this to take effect.
fetch https://mirrors.ustc.edu.cn/alpine/edge/main/x86_64/APKINDEX.tar.gz
fetch https://mirrors.ustc.edu.cn/alpine/edge/community/x86_64/APKINDEX.tar.gz
fetch https://mirrors.nju.edu.cn/alpine/edge/main/x86_64/APKINDEX.tar.gz
fetch https://mirrors.nju.edu.cn/alpine/edge/community/x86_64/APKINDEX.tar.gz
fetch https://mirrors.aliyun.com/alpine/edge/main/x86_64/APKINDEX.tar.gz
fetch https://mirrors.aliyun.com/alpine/edge/community/x86_64/APKINDEX.tar.gz
fetch https://mirror.lzu.edu.cn/alpine/edge/main/x86_64/APKINDEX.tar.gz
fetch https://mirror.lzu.edu.cn/alpine/edge/community/x86_64/APKINDEX.tar.gz
fetch https://mirrors.tuna.tsinghua.edu.cn/alpine/edge/main/x86_64/APKINDEX.tar.gz
fetch https://mirrors.tuna.tsinghua.edu.cn/alpine/edge/community/x86_64/APKINDEX.tar.gz
fetch https://mirrors.zju.edu.cn/alpine/edge/main/x86_64/APKINDEX.tar.gz
fetch https://mirrors.zju.edu.cn/alpine/edge/community/x86_64/APKINDEX.tar.gz
fetch https://mirrors.sjtug.sjtu.edu.cn/alpine/edge/main/x86_64/APKINDEX.tar.gz
fetch https://mirrors.sjtug.sjtu.edu.cn/alpine/edge/community/x86_64/APKINDEX.tar.gz
fetch https://dl-cdn.alpinelinux.org/alpine/edge/main/x86_64/APKINDEX.tar.gz
fetch https://dl-cdn.alpinelinux.org/alpine/edge/community/x86_64/APKINDEX.tar.gz

**Comment by kkkgo** [comment]
确实没有启动，你可以进容器内部执行`thttpd -p 7889 -d /data`看看是否有报错吗？
另外如果是为了配合PaoPaoGateway使用，可以用docker生成iso内嵌配置。

**Comment by kkkgo** [comment]
新版本已发布，`HTTP_FILE`功能已经使用mosdns内置的方式实现，可以使用新版本再试试。查阅更新日志：https://github.com/kkkgo/PaoPaoDNS/discussions/28
